backend/development.ini.sample

```diff
@@ -178,12 +179,30 @@ workspace.join.max_messages_history_count = -1
 ; sqlalchemy.url = sqlite:///%(here)s/tracim.sqlite
 sqlalchemy.url = %(basic_setup.sqlalchemy_url)s
 
-### File Depot ###
-## file depot name
-; depot_storage_name = tracim
-## This is where none database file will be stored
-; depot_storage_dir = %(here)s/depot
-depot_storage_dir = %(basic_setup.depot_storage_dir)s
+### Uploaded files ###
+
+; uploaded_files.storage_name = tracim
+## storage type, can be either "local" (local dir), "memory" (ram storage) or "s3" (remote S3 compatible storage)
+uploaded_files.storage_type = %(basic_setup.depot_storage_type)s
+
+## This is where none database file will be stored if storage_type is "local"
+uploaded_files.local.storage_path = %(basic_setup.depot_storage_dir)s
```

Comment:
I not realy understand this part and what is workflow to configure local, ram or s3 storage.
If storage_type is ram, is it managed by depot ?

Reply:
yes, ram/memory is managed by depot too. storage_path is a parameter only for "local" storage type (managed by depot), in case of s3 or memory, there is no need to give a local dir path, as this will not be the place where data will really be stored.